src/test/robot/api/storage/pravega.robot

```diff
@@ -47,6 +50,18 @@ Create Byte Streams Test
     Log  ${std_out}
     Validate Metrics Total Log File  ${step_id}  CREATE  ${count_limit}  0  104857600
 
+Create and read Events Test
```

Comment:
please rename to Read Events Test

Reply:
Please add some concurrency, 10 for example

Reply:
done